tests/integration/targets/helm_pull/tasks/main.yml

```diff
@@ -180,6 +180,7 @@
               - '"--username ansible" in _result.command'
               - '"--password ***" in _result.command'
               - '"--keyring pubring.gpg" in _result.command'
+              - '"Module did not set no_log for pass_credentials" not in _result.warnings'
```

Comment:
should be `_result.stderr` instead of `_result.warnings`, module helm_pull returns warnings into stderr

Reply:
Updated as requested